source/core/ut_suite_manager.pkb

```diff
@@ -327,5 +339,166 @@ create or replace package body ut_suite_manager is
     run_schema_suites(sys_context('userenv', 'current_schema'), l_temp_reported, a_force_parse_again);
   end run_cur_schema_suites_static;
 
+  function get_schema_suites(a_schema_name in varchar2) return tt_schema_suits is
+  begin
+    if not g_schema_suites.exists(a_schema_name) then
+      config_schema(a_schema_name);
+    end if;
+  
+    return g_schema_suites(a_schema_name);
+  end get_schema_suites;
+
+  -- Validate all paths are correctly formatted
+  procedure validate_paths(a_paths in ut_varchar2_list) is
+    l_path varchar2(32767);
+  begin
+    if a_paths is null or a_paths.count = 0 then
+      raise_application_error(ut_utils.gc_path_list_is_empty, 'Path list is empty');
```

Comment:
though about it but for now decided not to do it as it get much more difficult to figure out what is really called. the path will always be in form `suite1.suite2.test` and we have to understant wether its a suitepath of current schema ot suite1 is the schema name. Guessing mechanism is not yet implemented